partitioned-heat-conduction/openfoam-neumann/clean.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e -u
+
+. ../../tools/cleaning-tools.sh
+
+clean_openfoam .
```

Comment:
This tutorial is special in the sense that it first copies the `0.orig` to `0`. Let's clean this `0/` here (and in the openfoam-dirichlet):

```suggestion
clean_openfoam .
rm -rfv 0
```

Similarly, you can add a `clean.sh` in `openfoam-solver` that executes `wclean`. Don't forget to mark it as executable.

I noticed this as I am usually running `git status` after running and after cleaning and check for residue files.

Reply:
Done, but the `wclean` will not remove the solver executable (FYI)

Reply:
At least it removes the intermediate files from the `openfoam-solver/Make/<platform>` directory.